Doc/library/re.rst

```diff
@@ -1093,13 +1093,13 @@ Functions
 Exceptions
 ^^^^^^^^^^
 
-.. exception:: error(msg, pattern=None, pos=None)
+.. exception:: PatternError(msg, pattern=None, pos=None)
 
    Exception raised when a string passed to one of the functions here is not a
-   valid regular expression (for example, it might contain unmatched parentheses)
-   or when some other error occurs during compilation or matching.  It is never an
-   error if a string contains no match for a pattern.  The error instance has
-   the following additional attributes:
+   valid regular expression (for example, it might contain unmatched
+   parentheses) or when some other error occurs during compilation or matching.
+   It is never an error if a string contains no match for a pattern.  The
```

Comment:
These three lines: is the only change re-wrapping? If so, please undo. 

We generally prefer to avoid this: 
* it makes the diff bigger and harder to review
* it's more prone to conflicts in the future

Otherwise this PR looks ready.